dataGenerator.js

```diff
@@ -105,14 +105,38 @@ for (let index = 0; index < NUMOFHEADS; index++) {
   data.headmaster.push(fakeHeadmaster);
 }
 
-//menteess-----------------------------------------------------
+//Teachers-----------------------------------------------------
+data.teacher = [];
+for (let index = 0; index < NUMOFTEACHERS; index++) {
+  //Generate data
+  let fakeTeacher = {
+    id: index,
+    first_name: faker.name.firstName(),
+    last_name: faker.name.lastName(),
+    account_status: faker.random.boolean(),
```

Comment:
Wondering if we should change the data type of account status to a string so we can have more than two statuses, e.g. "Approved", "Denied", "Pending". I am thinking this will make it easier to filter out students and teachers who have applied previously and been denied versus those who are just pending.

Reply:
I agree that this will future-proof the account_status field. I will add a commit with the requested changes.